### ISSUE TITLE
Editing the registration form's gender field is not reflected in the FP Visit form

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-fp#47]
Steps to replicate:

- Register a new client, and put gender = other (biological sex can be either female or male)
- Start a new FP visit and go to the client history page. The method at intake is not displayed (this is a known issue and being addressed separately).
- Exit the FP visit without submitting
- Re-open the registration form and edit the person's gender from "other" to "female"
- Save the reg form and then start a new FP visit form.
- The gender edit does not reflect in the FP Visit form, because the method at intake question is still missing in the Client History page.

**Comment by cafootitt** [comment]
QA 14 May release - 

Unfortunately, I'm still unable to get this to work.

- I registered a person (May Jay, ID: 1747013), and originally made her gender = other and sex = female. The FP Visit form correctly showed the female methods for the Method at Intake section.
- I exited out of the FP visit form, then edited the registration form and changed the gender from other to male. I saved the reg form.
- I opened the FP visit form again. The methods listed under the method at intake question are still the female methods, not the male methods.
